src/Utils.js

```diff
@@ -10,6 +10,10 @@ import {
   REACT15,
 } from './version';
 
+function equality(a, b) {
```

Comment:
Ok, I will. 

Actually it's just a rewrite of the actual code (located [here](https://github.com/airbnb/enzyme/blob/master/src/Utils.js#L98))